.github/workflows/tests.yml

```diff
@@ -14,10 +14,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: Set up Ruby 2.6.5
```

Comment:
CI was broken

Reply:
CI is fixed now, right? Should we revert this

Reply:
GitHub actions doesn't support the 2.6.5 version of ruby. So I had to fix the tests.yml file to make CI pass. Nothing to revert here.